charts/osm/README.md

```diff
@@ -104,6 +104,7 @@ The following table lists the configurable parameters of the osm chart and their
 | OpenServiceMesh.osmcontroller.resource.requests.memory | string | `"128M"` |  |
 | OpenServiceMesh.outboundIPRangeExclusionList | list | `[]` | Optional parameter to specify a global list of IP ranges to exclude from outbound traffic interception by the sidecar proxy. If specified, must be a list of IP ranges of the form a.b.c.d/x. |
 | OpenServiceMesh.prometheus.port | int | `7070` | Prometheus port |
+| OpenServiceMesh.prometheus.resources | object | `{"limits":{"cpu":1,"memory":"2G"},"requests":{"cpu":0.5,"memory":"512M"}}` | Resource limits for prometheus instance |
```

Comment:
Not sure why this is looking like this.... I ran `make chart-readme`, not sure i'm missing something

Reply:
It picks up the default values defined in `values.yaml`. It looks fine, were you expecting something else?

Reply:
At line [101](https://github.com/openservicemesh/osm/pull/3181/files/491e6e77e0c7e99456ce7ce842b1e5a41c87177c#diff-5b88a1fc4bc9f14c08553477ec45b3628356876507a5cbe7a597abfc103db7daR101) seems to unroll the entire structure for `osmcontroller`, but not for prometheus here. Just wondered.